### PR TITLE
feat(list/bullet): add smallFont prop

### PR DESCRIPTION
**Pull request by JoanClaret**
Adding smallFont prop so we can reuse this component in different pages.

### CODE REVIEW

File: components/list/bullet/src/index.js [hunk 1]
```diff
@@ -5,9 +5,10 @@ import cx from 'classnames'
 
 const BASE_CLASS = 'sui-ListBullet'
 
-const ListBullet = ({listItems, responsive}) => {
+const ListBullet = ({listItems, responsive, smallFont}) => {
   const listBulletClass = cx(BASE_CLASS, {
-    [`${BASE_CLASS}--responsive`]: responsive
+    [`${BASE_CLASS}--responsive`]: responsive,
```
Comment (rmoralp) [inline components/list/bullet/src/index.js]: Are two variants compatible?

Comment (JoanClaret) [reply components/list/bullet/src/index.js]: We does not have specification about it  :/

File: components/list/bullet/src/index.scss [hunk 2]
```diff
@@ -30,12 +30,18 @@ $sz-list-bullet-illustration-l: 72px !default;
       #{$prefix}-illustration {
         height: $sz-list-bullet-illustration-l;
         margin-bottom: $m-xl;
-        margin-right: 0;
```
Comment (AgonisticKatai) [inline components/list/bullet/src/index.scss]: We need this specification to work properly on desktop-responsive

Comment (JoanClaret) [reply components/list/bullet/src/index.scss]: fixed. well seen mate.

**Pull request merged**